src/cljs/main/broadfcui/page/workspace/summary/tab.cljs

```diff
@@ -127,14 +128,14 @@
                                :text "Clone..." :icon :clone
                                :disabled? (when (empty? billing-projects) (comps/no-billing-projects-message))
                                :onClick #(modal/push-modal
-                                          [WorkspaceCloner
-                                           {:on-success (fn [namespace name]
-                                                          (swap! state dissoc :cloning?)
-                                                          (on-clone (str namespace ":" name)))
-                                            :workspace-id workspace-id
-                                            :description description
-                                            :is-protected? isProtected
-                                            :billing-projects billing-projects}])}])
+                                           [WorkspaceCloner
+                                            {:on-success (fn [namespace name]
+                                                           (swap! state dissoc :cloning?)
```

Comment:
might've changed spacing incorrectly (I just re-downloaded intellij and I'm not convinced all the settings are right, sorry)